README.rst

```diff
@@ -13,6 +13,12 @@ Hypothesis
 .. image:: https://img.shields.io/badge/IRC-%23hypothes.is-blue.svg
    :target: `#hypothes.is`_
    :alt: #hypothes.is IRC channel
+.. image:: https://img.shields.io/badge/license-BSD-blue.svg
+   :target: https://github.com/hypothesis/h/blob/683002c885ffa366cea33900aac396359b8f24d4/LICENSE
+   :alt: License badge
+.. image:: https://img.shields.io/badge/python-2.7-blue.svg
+   :target: https://github.com/hypothesis/h/blob/683002c885ffa366cea33900aac396359b8f24d4/tox.ini#L2
```

Comment:
I would remove this link, there's no point to link to the `tox.ini` for the Python version.